0.33.4 resolves #62

1jamesthompson1 opened the pull request:
TimeSeiresPlot() now works with ggplot.

Updated the help files explaining a bit about it.

All you can do regarding aesthetics is add titles. This does enough I believe for it to be useful. However, somewhat forcing students to use ggplot if they want more specific graphs seems like a good progression.

1jamesthompson1 committed:
0.33.4 resolves #62

TimeSeiresPlot() now works with ggplot.

ajrgodfrey commented:
Agreed, but we may need the command for the accessible SVG version.


From: 1jamesthompson1 ***@***.***>
Sent: Tuesday, 6 December 2022 12:51 pm
To: ajrgodfrey/BrailleR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ajrgodfrey/BrailleR] 0.33.4 resolves #62 (PR #65)


TimeSeiresPlot() now works with ggplot.

Updated the help files explaining a bit about it.

All you can do regarding aesthetics is add titles. This does enough I believe for it to be useful. However, somewhat forcing students to use ggplot if they want more specific graphs seems like a good progression.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/ajrgodfrey/BrailleR/pull/65<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=AfeU6L8eK4G7NjU8%2BGQZFB1ydm0%2BkRgmzOcpusvAWuk%3D&reserved=0>

Commit Summary

  *   311e900<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65%2Fcommits%2F311e9005731b4a258d59bca30d43c93b916c8a34&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ljpP090YsjhfxOLLSUqgmdbWCnY%2FMvKNXBa%2F31xCZRU%3D&reserved=0> 0.33.4 resolves #62

File Changes

(3 files<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65%2Ffiles&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=uleiy2%2F9nc1NS0PkyYYdt1vIJJFutdulwiLxkKNNPPE%3D&reserved=0>)

  *   M NEWS.md<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65%2Ffiles%23diff-51920e95310ebfbc1ae31709f3b95f89afffbf4f1a6e38e8b2b406e2fb6197ea&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FJ9mXJwBSEk%2B%2B21eQV5IV947VktrZBxfU36gWA%2BySaE%3D&reserved=0> (1)
  *   M R/TSPlot.R<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65%2Ffiles%23diff-353873f5258b5a57539f6d577a50a67f3b882f0d67a8f573fc8f7d577f115b34&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=mvlycngczaHAsMGwm9zYHnAPu4Eu8%2F98sK4Wvfjz9lI%3D&reserved=0> (51)
  *   M man/TSPlot.Rd<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65%2Ffiles%23diff-44c76b72440bb9715a92657feac8371feed391058f97688c2143ef981136d02a&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=vgukEM1xAskwQxW1re11yQKkMA5Q1qPFO%2BYE8UmB0g4%3D&reserved=0> (22)

Patch Links:

  *   https://github.com/ajrgodfrey/BrailleR/pull/65.patch<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65.patch&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=88egMkvwewfL%2Fe%2F56%2BBb15lL%2BB4xeyqSYIDuEcXSX54%3D&reserved=0>
  *   https://github.com/ajrgodfrey/BrailleR/pull/65.diff<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65.diff&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396211365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ch%2BVTtEMlsO4mH9nzk11pH%2F8Z%2Bz6pnkLrXAgbXz7XcE%3D&reserved=0>

-
Reply to this email directly, view it on GitHub<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F65&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396367610%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ybLhxr1KhPLAPhcmr4bkFoXzXxTRv3AtG0tHzUCaXvU%3D&reserved=0>, or unsubscribe<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FADCZKGUSP6IZB4N74JHQIVLWLZ5UVANCNFSM6AAAAAASU3N4YQ&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C1b9458043b5241bb97fe08dad71b8188%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058810396367610%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=l%2FQOPdzjMjSDFfVVXUfXpalyw3Ckv3eSxiW0yzWdayU%3D&reserved=0>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

1jamesthompson1 committed:
Merge branch 'master' into 0.33.4-issue-62-timeseries

1jamesthompson1 committed:
0.33.4 Fix build errors with TSPlot

Fixed errors with RD help file
Added imports to the NAMESPACE
Update tsplot code a bit for it to work with ts data correctly and handle the labels and title correctly.

1jamesthompson1 committed:
Merge branch 'master' into 0.33.4-issue-62-timeseries
